pkg/services/container.go

```diff
@@ -64,13 +64,20 @@ var Container = []dingo.Def{
 					cfg.RootCAs.AppendCertsFromPEM(ca)
 					klog.V(4).Infof("Added CA certificate %s", caPath)
 				}
+                tlsClientCert := os.Getenv("RABBITMQ_TLS_CLIENT_CERT")
+                tlsClientKey := os.Getenv("RABBITMQ_TLS_CLIENT_KEY")
+
+                if tlsClientCert != "" {
+                    if cert, err := tls.LoadX509KeyPair(tlsClientCert, tlsClientKey); err == nil {
```

Comment:
If this errors, a connection will be tried anyway, right?

Reply:
yes

Reply:
Should we notice the user about it?

Reply:
Do you mean to add a notice to the logs?